### PR TITLE
Templates (issue and PR) and Sonarcloud

### LINKED ISSUE #2: Sonarcloud analysis

### LINKED ISSUE #9: Issue template (feature/bug/other request)

### LINKED ISSUE #10: PR template

### PR DESCRIPTION
**Pull request by Tioborto**

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
 sonar.projectKey=Tioborto_raddar
 
 # relative paths to source directories. More details and properties are described
 # in https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/ 
-sonar.sources=.
\ No newline at end of file
+sonar.sources=.
+
+sonar.projectKey=Tioborto_raddar
+sonar.organization=tioborto
```
Comment (nymous) [inline sonar-project.properties]: You defined this key twice ^^

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+
+---
+<!-- Issues are for bugs and feature requests** only -->
```
Comment (nymous) [inline .github/ISSUE_TEMPLATE/bug_report.md]: What do the `**` mean?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+<!-- What did you expect to happen when running the steps above? -->
+
+##### ACTUAL RESULTS
+
+<!-- What actually happened? -->
+
+##### ADDITIONAL INFORMATION
+
+<!-- Include any links to sosreport, database dumps, screenshots or other
```
Comment (nymous) [inline .github/ISSUE_TEMPLATE/bug_report.md]: `sosreport`? We use RedHat stuff in this project? :stuck_out_tongue_closed_eyes:

**Review by nymous — CHANGES_REQUESTED** [review]
Veeeeery minor stuff but :+1:

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (Tioborto) [reply .github/ISSUE_TEMPLATE/bug_report.md]: It's bold. I forget "**" in the beginning

Comment (Tioborto) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I forget twice * in the beginning. It's a bold comment

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (Tioborto) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I change the comment

**Review by nymous — APPROVED** [review]

**Pull request merged**